nvm.sh

```diff
@@ -1698,10 +1699,16 @@ nvm_install_binary() {
   local TMPDIR
   local VERSION_PATH
 
+  local PROGRESS_BAR
   local NODE_OR_IOJS
   if [ "${FLAVOR}" = 'node' ]; then
     NODE_OR_IOJS="${FLAVOR}"
   fi
+  if [ "${NVM_NO_PROGRESS-}" = "1" ]; then
+    PROGRESS_BAR="-sS"
```

Comment:
should this be `--silent --show-error`, or would that make combining them tricky?

Is there a reason we ever don't want `--show-error` semantics?

Reply:
Yeah, let me change to `--silent --show-error`

Reply:
I think it's because we use parameters like `-L`, `-I`, `-C`, so it's to maintain consistency to use a short parameter and maybe I should even use `-#` instead of `--progress-bar`?

Reply:
As long as both forms work on the same systems, I'd prefer the long versions for readability.

Reply:
Maybe revise this somewhere else? Consistency is important.

Reply:
Fair. We can stick to the short versions now but please add the long ones in a comment.

Reply:
I mentioned this before (can't remember where).  The long option is not always available in some versions of `curl`.  However, the short option definitely is.  For compatibility, I recommend sticking to the short option for this specific feature.

Reply:
Thanks, in that case let's stick with the short options (but comment the long ones, for clarity)